images/php-cli/entrypoints/61-php-xdebug-cli-env.sh

```diff
@@ -4,7 +4,7 @@
 if expr "$XDEBUG_ENABLE" : '[Tt][Rr][Uu][Ee]' > /dev/null; then
   # XDEBUG_CONFIG is used by xdebug to decide if an xdebug session should be started in the CLI or not.
   # The content doesn't really matter it just needs to be set, the actual connection details are loaded from /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
-  export XDEBUG_CONFIG="idekey=lagoon"
+  export XDEBUG_SESSION="lagoon"
```

Comment:
This is correct according to the [upgrade guide](https://xdebug.org/docs/upgrade_guide), but from my reading `XDEBUG_SESSION` is a [legacy fallback](https://xdebug.org/docs/all_settings#start_with_request) and the new key to use is `XDEBUG_TRIGGER`. I've tested this locally, so I think it makes more sense to change this to `export XDEBUG_TRIGGER="cli"`.

Would you also update the comment?